src/components/flowchart-wrapper/flowchart-wrapper.js

```diff
@@ -119,7 +119,13 @@ export const FlowChartWrapper = ({
       const modularPipeline = nodes[nodeId];
       const hasModularPipeline = modularPipeline?.length > 0;
 
-      if (hasModularPipeline) {
+      const isParameterType =
+        graph.nodes &&
+        graph.nodes.find(
+          (node) => node.id === nodeId && node.type === 'parameters'
+        );
+
+      if (hasModularPipeline && !isParameterType) {
```

Comment:
Could you explain why this is needed again? You mentioned it earlier though now I've forgot.

Reply:
hey because both parameters and some nested nodes can both have a modular pipeline as a parent. therefore it needs to check that the node is not a parameter type before it tries to expand the modelarPipeline. 

I'll include a comment here to explain this 👍